task.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir -p ~/task/dir1 ~/task/dir2 ~/task/dir3/dir4
+
+# изменяем текущую директорию на task
+cd ~/task
+
+# создаём пустой файл task/dir2/empty
+touch ~/task/dir2/empty.txt

```

Comment:
Мы уже в папке `tasks`, можно ее не указывать
```suggestion
touch dir2/empty.txt
```

Reply:
нужно далее поправить